composer.json

```diff
@@ -37,6 +37,7 @@
     "paypal/rest-api-sdk-php": "*",
     "predis/predis": "~1.0",
     "sentry/sentry-laravel": "*",
+    "symfony/finder": "*",
```

Comment:
this probably isn't needed. If laravel changes the return, it'll explode in different way anyway and this wouldn't help ┐(°～° )┌